data/harvesting.txt

```diff
@@ -8,6 +8,28 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
+minable "mereti_debris"
```

Comment:
Remove the underscore. The minable names in the maps can simply be in quotes, e.g. `minables "sestor debris" 2 7.17656`

Reply:
Done